trusted_os/handler.s

```diff
@@ -15,6 +15,12 @@
 #include "go_asm.h"
 #include "textflag.h"
 
+#define g_timer				208
```

Comment:
worth pre/suf-fixing these with `pre_go122` to make it super clear   what they're for?

Reply:
I see these as temporary solution as they are real ugly magic numbers anyway, to avoid clutter what about moving them within the function (without passing through #defines) and commenting them?

Reply:
I think `#defines` are nice/make it easier to read, so I'd leave like this.

My main thought here was to make it super clear why these defines exist, so when someone comes along to rip out the legacy wakeG func they know to take these out too.

I don't feel super strongly about it, though.

Reply:
Resolved in https://github.com/transparency-dev/armored-witness-os/pull/284/commits/af57ab62c4d2dc5036f8c8017ac3e0192ea87441